libs/librepcb/project/schematics/items/si_netlabel.cpp

```diff
@@ -81,6 +81,12 @@ NetSignal& SI_NetLabel::getNetSignalOfNetSegment() const noexcept {
   return mNetSegment.getNetSignal();
 }
 
+Length SI_NetLabel::getWidth() noexcept {
+  Length width = Length::fromPx(mGraphicsItem->boundingRect().width());
```

Comment:
`boundingRect().width()` also contains the origin cross on the left side, so it is too wide. Probably `boundingRect().right()` would be better. And the result can be returned without creating a local variable first ;)

```suggestion
  return Length::fromPx(mGraphicsItem->boundingRect().right());
```

Reply:
I have a habit putting intermediate variables (simplifies debugging a lot when putting breakpoint for the very exact point). Optimized by compiler code usually is the same since temporary variables created anyway. Just personal preferences then.